tests/src/JIT/Directed/intrinsic/pow/pow1.cs

```diff
@@ -14,16 +14,18 @@ public static int Main()
         double x, y, z;
         bool pass = true;
 
-        //Check if the test is being executed on ARM
-        bool isProcessorArm = false;
+        //Check if the test is being executed on ARMARCH
+        bool isProcessorArmArch = false;
 
         string processorArchEnvVar = null;
 
         processorArchEnvVar = Environment.GetEnvironmentVariable("PROCESSOR_ARCHITECTURE");
 
-        if ((processorArchEnvVar != null) && processorArchEnvVar.Equals("ARM", StringComparison.CurrentCultureIgnoreCase))
+        if ((processorArchEnvVar != null)
```

Comment:
It's good suggestion. But OSArchitecture or ProcessArchitecture under RuntimeInformation relies on GetNativeSystemInfo from OS which doesn't appear to be fully implemented yet on ARM64. This was okay for x64/Windows, but when I tried this on ARM64, I got null object. So, I think I will keep this for now as it is.